features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/mapper/BaseWireframeMapper.kt

```diff
@@ -180,7 +180,7 @@ abstract class BaseWireframeMapper<T : View, S : MobileSegment.Wireframe>(
             y = bounds.y,
             width,
             height,
-            view.background,
+            view.background?.constantState?.newDrawable(),
```

Comment:
why do you create a new drawable here ? isn't this costly for each view ?

Reply:
preferably we wouldn't need to do this, however if the drawable is not moved to a new instance before we create the bitmap, then drawing the bitmap on the canvas modifies the view in the app itself.

Reply:
ok I see so you need to take a copy ...hmmm...I wonder how expensive this is for each view. Sorry for asking but why do you need to modify the Drawable to create the bitmap from it ?

Reply:
I need to set the bounds before I can draw to the canvas

Reply:
I'd say that if profiling it shows decent number we can go with this like that. @0xnm, @xgouchet waiting for your thoughts on this one. I searched for a way to go around it, but apparently there's not, the only other way is to `setBounds` when before `drawing` and try to put the drawable back into its previous state after. Not sure how reliable this is and honestly I think I would rather go with the copy.